Common/Tests/Utilities.UI/UI/ReplWindowProxy.cs

```diff
@@ -349,6 +411,22 @@ string executionMode
                 .Select(l => l.TrimEnd('\r', '\n', ' '))
                 .ToList();
 
+#if DEV14_OR_LATER
+            dynamic eval = _window.Evaluator;
+            var primary = eval.PrimaryPrompt as string ?? ">>>";
+#else
```

Comment:
For this particular case, it's under `Common`, so we really ought to be minimizing breaking changes here (to help the other people who are using this directory for their own projects). That said, I'm not sure the repl window proxy is helpful to anyone right now, so maybe it's best to just rip it out and put it into one of our Python-specific test assemblies?